netbout-web/src/main/java/com/netbout/email/EmMessages.java

```diff
@@ -93,6 +93,13 @@
         this.self = slf;
     }
 
+    /**
+     * Post a new message.
```

Comment:
@erimerturk no need for javadoc here. for the puzzle, use multiple `//` comments